packages/compass-e2e-tests/helpers/commands/click-visible.ts

```diff
@@ -0,0 +1,12 @@
+import { Browser } from 'webdriverio';
```

Comment:
I think this can usually be 

```suggestion
import type { Browser } from 'webdriverio';
```

correct? That’s often a bit nicer because it differentiates a type-only dependency from a dependency on the code (and helps understand the structure of the code that way)

Reply:
Makes sense. I just didn't know about import type. Will change.